cmd/fleetctl/api.go

```diff
@@ -81,14 +82,18 @@ func clientFromCLI(c *cli.Context) (*service.Client, error) {
 	}
 
 	// check that AppConfig's Apple BM terms are not expired.
-	appCfg, err := fleetClient.GetAppConfig()
-	if err != nil {
+	var sce kithttp.StatusCoder
```

Comment:
Is this change related to the Gitops task? 
Or you fixed it on the fly?

Reply:
Related to the GitOps task.

(See the comment below. Basically GitOps doesn't have access to the app's config so we need to support that case.)